Files.cs

```diff
@@ -618,9 +618,9 @@ public static bool LoadConfig()
             StreamReader str = null;
             try
             {
-                if (File.Exists(Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.ApplicationData), "TQ Collector Release\\config.xml")))

```

Comment:
I thought it might be easier to have the config.xml file in the same folder as the executable, thus making TQCollector portable. Should have been its own commit, but I messed that up a bit.

Reply:
Ok. The original idea was that the AppData folder is where Windows has designated for modifiable files to go, but you can move it if you want.